src/app.jsx

```diff
@@ -144,3 +138,70 @@ export class Application extends React.Component {
         );
     }
 }
+
+const DropdownWithKebab = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const onToggleClick = () => {
+        setIsOpen(!isOpen);
+    };
+    const onSelect = (_event, itemId) => {
+        setIsOpen(false);
+    };
+    return (
+        <Dropdown
+            isPlain
+            isOpen={isOpen}
+            onSelect={onSelect}
+            onOpenChange={isOpen => setIsOpen(isOpen)}
+            toggle={toggleRef =>
+                <MenuToggle ref={toggleRef} aria-label="kebab dropdown toggle" variant="plain" onClick={onToggleClick} isExpanded={isOpen}>
+                    <EllipsisVIcon />
+                </MenuToggle>}
+        >
+            <DropdownList>
+                <DropdownItem itemId={0} key="action1">
+                    Action
+                </DropdownItem>
+                <DropdownItem itemId={1} key="action2">
+                    Action
+                </DropdownItem>
+                <Divider component="li" key="separator" />
+                <DropdownItem itemId={2} key="action3">
+                    Action
+                </DropdownItem>
+            </DropdownList>
+        </Dropdown>
+    );
+};
+
+export const DropdownBasic = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const onToggleClick = () => {
```

Comment:
When one liner you can avoid the code block.

const onToggleClick = isOpen => setIsOpen(!isOpen);